container.go

```diff
@@ -239,41 +218,55 @@ func terminate(profile *Profile, all bool) error {
 	} else {
 		f.Add("label", "com.viam.canon.profile="+profile.name+"/"+profile.Arch)
 	}
-	containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{Filters: f})
+	containers, err := cli.ContainerList(ctx, container.ListOptions{All: true, Filters: f})
```

Comment:
Probably just my ignorance: if you give it a filter but you also set All to true, do you get all of them, or just the ones that match the filter? I wonder if this line was changed during debugging and forgot to be changed back. but maybe it's correct as-is?

Reply:
All means "include stopped containers"... it's annoyingly worded. Same as if you do `docker ps` you won't get stopped containers without the `--all` flag there.